eq-author/src/App/AccessDeniedPage/index.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { render } from "tests/utils/rtl";
+
+import AccessDenied from "./";
+
+describe("Access denied page", () => {
+  beforeEach(() => {
+    const div = document.createElement("div");
```

Comment:
This is resolved by the updated `render` so this should not be necessary

Reply:
Now using the new render.